.changeset/quick-cougars-fry.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-home': minor
```

Comment:
```suggestion
'@backstage/plugin-home': patch
```
^Could you update this changeset to a `patch` change since this is a non breaking change?
https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#how-to-create-a-changeset

Reply:
Woops, done!